src/desktop/apps/article/components/InfiniteScrollNewsArticle.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { Component } from 'react'
+import { flatten } from 'lodash'
+import { Article } from '@artsy/reaction/dist/Components/Publishing'
+import { ArticleData } from '@artsy/reaction/dist/Components/Publishing/Typings'
+import Waypoint from 'react-waypoint'
+import styled from 'styled-components'
+import { positronql as _positronql } from 'desktop/lib/positronql'
+import { newsArticlesQuery } from 'desktop/apps/article/queries/articles'
+import { setupFollows, setupFollowButtons } from './FollowButton.js'
+
+export interface Props {
+  article: ArticleData
+  isMobile: boolean
+  marginTop: string
+}
+
+interface State {
+  isLoading: boolean
+  articles: ArticleData[]
+  offset: number
+  error: boolean
+  following: any[]
+  isEnabled: boolean
+}
+
+let positronql = _positronql
```

Comment:
Totally cool to do the rewire stuff in a different PR, but curious if its working...

Reply:
Tried enabling this today, it seems to be unhappy with requiring coffeescript files. When requiring `./src/desktop/assets/article.coffee` it returns the error:
`TypeError: Property typeName of TSTypeReference expected node to be of a type ["TSEntityName"] but instead got "CallExpression"`

Reply:
Let's go ahead and address this in another PR. I'm eager to build on `InfiniteScrollNewsArticle` 😄 . @eessex can you please comment a TODO here so we remember to address it?